### ISSUE TITLE
Only three stops on planning route ?

### ISSUE THREAD

**Issue by pawelbigus** [mapsme/omim#9995]
Can you increase the number of stops?
Three stops are definitely not enough to make my route well.
Is this possible to increase max numbers of stops ?

**Comment by mariusel09** [comment]
yes,he has right,can you increase number of stops up to 40-50,even 100 please?

**Comment by spacekookie** [comment]
Why is there a maximum number of stops anyways? 🤨

**Comment by davebrny** [comment]
ive been driving off the main roads lately recording street-view imagery for mapillary, so when im trying to plan a route it usually sends me back to the main road since that would be the quicker route or better road. adding stops along the way helps keep the route where i want but i dont always have enough

**Comment by burivuh** [comment]
Hi!

Limitation is on the UX/UI part. We will think about it.

Thanks!
